src/UI/Settings/Notifications/Edit/NotificationEditView.js

```diff
@@ -13,7 +13,8 @@ var view = Marionette.ItemView.extend({
     ui : {
         onDownloadToggle : '.x-on-download',
         onUpgradeSection : '.x-on-upgrade',
-        tags             : '.x-tags'
+        tags             : '.x-tags',
+        pushbullet       : '#x-pushbullet'
```

Comment:
Also not sure if this belongs as a selector for the whole view since its very specific to PushBullet.